TCMSApp/src/client/app/tests/tests.service.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.tests')
+        .service('TestsService', TestsService);
+
+TestsListController.$inject = ['FakeTestsFactory'];
```

Comment:
do you see the beauty? :)

now your controller will not change if we will substitute FakeTestsFactory to something real :)